README.md

```diff
@@ -0,0 +1,91 @@
+# Quiz-rs - Terminal quiz cards application
+
+Quiz-rs is a quiz card application built within the terminal.
+
```

Comment:
Add, "Credit: This application was heavily inspired by [hascard](https://github.com/Yvee1/hascard)"